tests/test_repository_tool.py

```diff
@@ -1676,6 +1711,27 @@ def test_revoke(self):
 
 
 
+  def test_check_path(self):
+    # Test that correct path does not raise exception: using '/' as a separator
+    # and does not start with a directory separator
+    self.targets_object._check_path('file1.txt')
+
+    # Test that non-exsitent path does not raise exception (_check_path works
+    # checks only the path string for compliance
+    self.targets_object._check_path('non-existent.txt')
+    self.targets_object._check_path('subdir/non-existent')
+
+    # Test improperly formatted rolename argument.
+    self.assertRaises(securesystemslib.exceptions.FormatError,
+        self.targets_object._check_path, 3)
+
+    # Test improperly formatted rolename argument.
```

Comment:
Same here, is this comment correct?

Reply:
Fixed in squashed commit https://github.com/theupdateframework/tuf/pull/1008/commits/3a13b7312a9148f7aac553affa371f3f26c69f11